src/dlgTriggerEditor.cpp

```diff
@@ -561,13 +561,20 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
         mProfileSaveAsAction->setToolTip(disabledSaving);
     }
 
-    auto *nextSectionShortcut = new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_Tab), this);
+    // a.k.a. QKeySequence::NextChild
+    //: Shortcut for next profile (tab) in main Mudlet application window
+    auto *nextSectionShortcut = new QShortcut(QKeySequence(tr("Ctrl+Tab")), this);
```

Comment:
Is it right to wrap this in tr()?

Reply:
Yeah, see https://doc.qt.io/qt-6/qkeysequence.html#keyboard-layout-issues also, for instance see this image from Wikipedia that shows a German DIN Standard layout - note that `Ctrl` is `Strig` there...

Reply:
That link says that QKeySequence(Qt::CTRL | Qt::Key_Tab) is a valid approach, what am I missing?

Reply:
It throws warnings in cc++20/cc+2a compilers in Qt 5.18 see [QTBUG-99948](https://bugreports.qt.io/browse/QTBUG-99948), which is still around even if WE are choosing to move to Qt 6.
![image](https://github.com/user-attachments/assets/4aa7763b-f9ff-44d5-868a-184bf3cee45b)

Reply:
I don't think it's a good idea to give more work to translators for something that is going away (Qt 5). 

All builds are using Qt 6 now and it's time to drop support for Qt 5 as we have no need for it.

Reply:
Using mixed `enums` is only working because Qt put in bodges (special constructors) for Qt6 to get around it - but the fixes only went into 5.15.9 which isn't out in the wild yet (the year is not up). Anyhow from an **I18n**/**L10n** point of view it would be better if the texts can be localised - what does @Kebap - as a German, who would know more about `Strig` vs `Ctrl` - think?

Reply:
Having dived in I've just found that at least some of the shortcuts in the `dlgTriggerEditor` **don't work** - `<Ctrl>+<Tab>` and `<Ctrl>+<BackTab>` {a.k.a. `<Ctlr>+<Shift>+<Tab>`} don't work to move between the different editor type view!

Also the one being discussed in https://github.com/Mudlet/Mudlet/pull/7638/#pullrequestreview-2596454283 does not do anything. The "Close profile" short-cut is recognised from the main window shortcut from a floating (but dockable) type "User window" but that seems to be because the event is passed up the ancestors to the main window and then to the main menu.

Reply:
It's not entirely clear what you're saying, but I see that the latest available Qt 6.8 release says that creating key combinations with a + is deprecated, while using  | as we do is fine, see documentation: https://doc.qt.io/qt-6/qkeysequence.html

Qt 5 issues are not a consideration as we'll remove Qt 5 support soon (we don't need it).

Reply:
@SlySven can you confirm that the current | syntax is allowed by the Qt documentation?

The point here is that we need to be mindful of volunteer translators time and cannot add more work unless we're sure it is necessary 😃

Reply:
This is academic for **this** PR as I've decided to forget about trying to fix the bodging to get `QShortcut`s to compile cleanly on any Qt version.

Reply:
Well, I'd prefer to be properly localisable in the long run - which does require the use of translatable strings as the source of the `QKeySequence`s - the matter hasn't gone away - it is just that I have pulled it from ***this*** PR.